docs/source/options.rst

```diff
@@ -0,0 +1,107 @@
+.. Copyright 2016 The Cartographer Authors
+
+.. Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+..      http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=======================
+Configuration Reference
+=======================
+
+Top-level options of the Cartographer ROS integration:
```

Comment:
Cartographer's ROS integration top-level options:

Add some text about Lua configuration? All values must be specified, etc.